fenix/app/metrics.yaml

```diff
@@ -33,7 +33,6 @@ events:
     notification_emails:
       - telemetry-client-dev@mozilla.com
       - android-probes@mozilla.com
-      - cgordon@mozilla.com
```

Comment:
Hmmm, where did all these other changes come from? I was expecting to see just the `fx_suggest:` section, but it looks like there are a lot of unrelated changes to the other metrics.

Reply:
ohhh hmm did i do a bad rebase, 😬, will fix this

Reply:
yup, got frustrated with figuring out why the yaml wasn't structured correctly, so decided to checkout the one from main... which is why it got weird. Should be fixed now